webpack.config.js

```diff
@@ -303,6 +303,12 @@ module.exports = env => {
           // split off into a seperate chunk, and named chunks will break (assume that's a bug that we can fix).
         },
       }),
+
+      // new Repack.plugins.ChunksToHermesBytecodePlugin({
```

Comment:
Not technically. It turns out we can build the app with hermes but load non-hermes bundles, so that's what I did for now. I still wanted to leave this in though for future use (potentially).